ethereum/test/test_eth_app.js

```diff
@@ -156,29 +172,88 @@ describe("ETHApp", function () {
 
       // decode event
       var iface = new ethers.utils.Interface(ETHApp.abi);
-      let event = iface.decodeEventLog(
+      let unlockEvent = iface.decodeEventLog(
         'Unlocked(bytes32,address,uint128)',
+        receipt.rawLogs[1].data,
+        receipt.rawLogs[1].topics
+      );
+
+      unlockEvent.recipient.should.be.equal(recipient);
+      unlockEvent.amount.eq(ethers.BigNumber.from(amount)).should.be.true;
```

Comment:
```suggestion
      unlockEvent.sender.should.be.equal(this.app.address);
      unlockEvent.recipient.should.be.equal(recipient);
      unlockEvent.amount.eq(ethers.BigNumber.from(amount)).should.be.true;
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/697/commits/9350f767fbc05395172d31650b60bbffa5591c42